WordPress/jetpack_metadata/release_notes.txt

```diff
@@ -1,8 +1,5 @@
-* [**] Video uploads: video upload is now limited to 5 minutes per video on free plans.
-* [*] Block editor: Give multi-line block names central alignment in inserter [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4343]
-* [**] Block editor: Fix missing translations by refactoring the editor initialization code [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4332]
-* [**] Block editor: Add Jetpack and Layout Grid translations [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4359]
-* [**] Block editor: Fix text formatting mode lost after backspace is used [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4423]
-* [*] Block editor: Add missing translations of unsupported block editor modal [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4410]
-* [***] My Site: Introduced cards based dashboard feed with posts card [https://github.com/wordpress-mobile/WordPress-Android/issues/15198]
-
+- Manage your dashboard with a card-based feed, including Posts.
+- Upload up to 5-minute videos on free plans.
+- See more user interface text in your language, including Jetpack/Layout Grid blocks.
+- Keep text formatting after hitting backspace.
+- Block titles are now center-aligned in the inserter menu.
```

Comment:
As mentioned in the Slack channel we have with our freelancer, I tweaked the editorial'd release notes compared to what she provided us:
 - I removed an item that she incorrectly included in the Android Release Notes (this was an item about timezone selection in iOS but not listed in Android)
 - I reworded her last bullet point as it would otherwise read oddly to me and misdescribe the change made by the corresponding PR

Also note that these editorial'd release notes don't include the last-minute entry about the "Share with Friends" being reintroduced on Me screen (see entry addition in `RELEASE-NOTES.txt` [just above](https://github.com/wordpress-mobile/WordPress-Android/pull/15821/files#diff-558ce8820a544da19f7fccdb2d1d661e46d68c130a1a39abdd222aa3484a5873R16)).